### PR TITLE
css/steps-scrollable

### LINKED ISSUE #1902: convert css for steps

### LINKED ISSUE #1903: convert css for starrating

### LINKED ISSUE #1904: convert css for select

### LINKED ISSUE #1905: convert css for searchinput

### LINKED ISSUE #1906: convert css for scrollable

### PR DESCRIPTION
**Pull request by EdwardIrby**
resolves #1902 
resolves #1903 
resolves #1904 
resolves #1905 
resolves #1906

### CODE REVIEW

**Review by sweeting — CHANGES_REQUESTED** [review]
⛔ Steps
Completed marker should be green `--pscolorsstatussuccess`
CleanShot 2021-07-08 at 08 27 22

✅ Star rating

✅ Select

⛔ Search Input
Loading isn't animated

✅ Scrollable

✅ AppFrame

File: packages/scrollable/src/react/index.tsx [hunk 1]
```diff
@@ -2,29 +2,12 @@ import {
   canUseDOM,
   combineFns,
   shallowCompare,
-  useResizeObserver
+  useResizeObserver,
+  classNames
```
Comment (jaketrent) [inline packages/scrollable/src/react/index.tsx:6]: nit: alpha order on imports is a project convention

File: packages/select/src/react/__specs__/index.spec.tsx [hunk 2]
```diff
@@ -34,6 +34,30 @@ describe('Select', () => {
     expect(ref).not.toBeNull()
   })
 
+  it('composes className for (Select/Select.Button', () => {
```
Comment (jaketrent) [inline packages/select/src/react/__specs__/index.spec.tsx:37]: nit: extra paren in description

File: packages/starrating/src/css/index.css [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+}
+.psds-starrating__star--active {
+  color: var(--ps-colors-yellow-6);
+}
+.psds-starrating__star--bright.psds-theme--dark {
+  color: var(--ps-colors-text-icon-low-on-dark);
+}
+.psds-starrating__star--bright.psds-theme--light {
+  color: var(--ps-colors-text-icon-low-on-light);
+}
```
Comment (jaketrent) [inline packages/starrating/src/css/index.css:38]: dunno if it's in this PR or not, but the different approaches to theming in selectors w/in the same component seems to be a code smell.

**Review by jaketrent — CHANGES_REQUESTED** [review]
Looks good. At least just one small change to cleanup.

File: packages/starrating/src/css/index.css (hunk 3 above)
Comment (EdwardIrby) [reply packages/starrating/src/css/index.css:38]: I agree!

**Review by jaketrent — APPROVED** [review]

**Review by sweeting — APPROVED** [review]
steps: completed marker looks good

**Pull request merged**